examples_tests/01.HelloWorld/main.cpp

```diff
@@ -11,6 +11,13 @@ nothing fancy, just to show that Irrlicht links fine
 #include <cstdio>
 #include <nabla.h>
 
+// Temporary
+#include <volk/volk.h>
+#include "../../src/nbl/video/CVulkanPhysicalDevice.h"
+#include "../../include/nbl/video/surface/ISurfaceVK.h"
+
+#include <nbl/ui/CWindowManagerWin32.h>
+#include <nbl/system/ISystem.h>

```

Comment:
if these dont come automatically from `nabla.h` (which includes `ui.h` and `system.h` they need to be added)

Reply:
There is no `system.h`! Got removed when I merge `declarations_and_definitions` into this branch.
Shall I put both of them in `ui.h`?

Reply:
> There is no system.h! Got removed when I merge declarations_and_definitions into this branch.

ok in this case, `nabla.h` includes `system/declarations.h` and `system/definitions.h` same pattern with ui

are the `CWindowManagerWin32` and `ISystem` not already in `ui/declarations.h`  and `system/declarations.h` respectively? (@sadiuk did ISystem get split into a declaration and definition?)

Reply:
umm.. there's no `ui/declarations.h`

`ISystem.h` is included in `system/declarations.h`

Reply:
well then `ISystem` shoudl be provided by `nabla.h`

and make a `ui/declarations.h` and `ui/definitions.h` out of `ui/ui.h`